%% The text of your abstract and nothing else (other than comments) goes here.
%% It will be single-spaced and the rest of the text that is supposed to go on
%% the abstract page will be generated by the abstractpage environment.
Microbial ecology has benefited from the decreased cost and increased quality
of next-generation DNA sequencing. In general, studies that use DNA sequencing
are no longer limited by the sequencing itself but instead by the acquisition
of the samples and by methods for analyzing and interpreting the resulting
sequence data.
In this thesis, I describe the results of three projects that address challenges
to interpreting or acquiring sequence data.
In the first project,
I developed a method for analyzing the dynamics of the relative abundance of
operational taxonomic units measured by next-generation amplicon sequencing in 
microbial ecology experiments without replication. In the second project, I
and my co-author combined a taxonomic survey of a dimictic lake, an ecosystem-level
biogeochemical model of microbial metabolisms in the lake, and the results of
a single-cell genetic assay to infer the identity of taxonomically-diverse,
putatively-syntrophic microbial
consortia. In the third project, I and my co-author developed a model of differences
in the efficacy that stool from different donors has when treating patients
via fecal microbiota transplant. We use that model to compute statistical
powers and to optimize clinical trial designs.
Aside from contributing scientific conclusions about each system, these
methods will also serve as a conceptual framework for future efforts to address
challenges to the interpretation or acquisition of microbial ecology data. 
